.storybook/main.ts

```diff
@@ -1,5 +1,13 @@
-import type { StorybookConfig } from '@storybook/core-common';
+import type { StorybookConfig } from '@storybook/angular';
 
+const frameworkName: '@storybook/angular' = '@storybook/angular';
```

Comment:
Storybook 7 introduced stronger typings to the `StorybookConfig`; however, their `name` type for the framework object is set to very strict strings and strict mode was yelling if I left it as `string` - so that is why the weird type is set here.